### ISSUE TITLE
Spherical_sld should not have "0 layers" as an option. (Trac #635)

### ISSUE THREAD

**Issue by ajj** [SasView/sasview#767]
Migrated from http://trac.sasview.org/ticket/635
```json
{
    "status": "closed",
    "changetime": "2018-11-21T13:56:17",
    "_ts": "2018-11-21 13:56:17.743318+00:00",
    "description": "",
    "reporter": "ajj",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-08-29T20:00:42",
    "component": "SasView",
    "summary": "Spherical_sld should not have \"0 layers\" as an option.",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 5.0.0",
    "owner": "",
    "type": "defect"
}
```

**Comment by pkienzle** [comment]
Trac update at `2016/08/30 16:19:53`: 
* **pkienzle** commented:

This should no longer generate errors on plotting unless background=0, so it is not a blocking issue and can be moved to the next release.

Either update fitting/basepage.py to allow more flexibility with the multiplicity combo box or update the model so that the parameters for the core are always present. 


* **pkienzle** changed milestone from "SasView 4.0.0" to "SasView Next Release +1"

**Comment by ajj** [comment]
Trac update at `2016/08/31 15:58:26`: 
* **ajj** commented:

Will this affect future backwards compatibility of saved projects/analysis? If so, it needs to be fixed before 4.0


* **ajj** changed milestone from "SasView Next Release +1" to "SasView 4.0.0"

**Comment by butlerpd** [comment]
Trac update at `2016/09/03 01:56:30`: 
* **butler** commented:

Have agreed to move the new models to next release at last conf call - hence moving it now.


* **butler** changed milestone from "SasView 4.0.0" to "SasView Next Release +1"

**Comment by butlerpd** [comment]
Trac update at `2017/10/27 11:25:07`: **butler** changed milestone from "SasView Next Release +1" to "SasView 5.0.0"

**Comment by rozyczko** [comment]
Trac update at `2018/11/21 13:55:55`: **piotr** commented:

This has been addressed in 5.0.

**Comment by rozyczko** [comment]
Trac update at `2018/11/21 13:56:17`: 
* **piotr** changed resolution from "" to "fixed"
* **piotr** changed status from "new" to "closed"
